docs/api.md

```diff
@@ -538,6 +538,9 @@ by the following errors
 * `code: 400, errno: 144`:
   Email already exists
 
+* `code: 400, errno: 151`:
+  Failed to send email
```

Comment:
Maybe a bit, but is this a 400-level error (client's fault) or a 500-level error (server's fault)?  Or maybe it can fail for both reasons and we should show different messaging for the two cases?